website/docs/reference/widgets/README.md

```diff
@@ -39,6 +39,7 @@ The following properties are common across many of Appsmith's widgets. You can f
 
 | Property            | Description                                                                                                                                                                                      |
 | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [**Height**](./README.md#height)        | It configures how a widget’s height reacts to content changes. It has three possible configurations:<br/>**Fixed**: The height of the widget remains as set using drag and resize.<br/>**Auto Height**: The height of the widget reacts to content changes.<br/>  **Auto Height with limits**: Same as Auto height, with a configurable option to set the minimum and maximum number of rows that can be occupied by the widget.                                      |
```

Comment:
Just give an anchor, and it should work. `#height`

Reply:
Any reason why we are giving Height at the top. I suggest we place all properties in the order they appear on the platform

Reply:
These properties may or may not be a part of properties. Common properties are a set of properties that are common in more than 5/6 widgets. In terms of order, we either go for alphabetical or order of importance. Here we put the height at the top as it's more important than other listed property.

Reply:
Hmmm. Even with the common properties, aren't we making the users hunt for which widgets have these properties. we should only keep the most common ones, like required, tooltip etc..

Reply:
All the widget pages have hyperlinks, so it's easy for users to land here.